methode-article-internal-components-mapper.yaml

```diff
@@ -2,11 +2,11 @@ server:
 
   applicationConnectors:
     - type: http
-      port: 11070
+      port: 11080
```

Comment:
Why did you changed this?

Reply:
i forgot, i was debugging while methode-article-mapper was still running on port 11070, maybe i can leave this?

Reply:
It's fine by me.